samples/subsys/usb/mass/README.rst

```diff
@@ -114,36 +147,6 @@ different):
 For information on mounting littlefs file system on Linux or FreeBSD
 systems refer to the "littlefs Usage" section below.
```

Comment:
This looks fine and works, but needs the patch at the top of https://github.com/pabigot/zephyr/commits/pr/30293 to accommodate the change in littlefs partition size.

Reply:
thanks, applied